src/pages/help-centre/components/_side-tab.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { TQuestions } from '../data/_data-types'
+import { Header } from 'components/elements'
+import device, { size } from 'themes/device'
+import { getWindowWidth } from 'common/utility'
+import { Desktop } from 'components/containers/show'
+import { useTabStateQuery } from 'components/hooks/use-tab-state-query'
+import { TString } from 'types/generics'
+import { Localize } from 'components/localization'
+
+export type TChildren = {
```

Comment:
```suggestion
interface ChildProps {
    label: string
}

class ChildComponent extends React.Component<ChildProps> {}

type SideTabType = {
    tab_header: TString
    data: TQuestions[]
    children: Array<ReactElement<ChildProps, JSXElementConstructor<ChildComponent>>>
}
```
just a suggestion, please check it out and let me know what you think, thanks :pray:

Reply:
I like this, will change that